src/librustc/ty/mod.rs

```diff
@@ -1332,17 +1334,6 @@ impl<'a, 'tcx> ParameterEnvironment<'tcx> {
 pub struct Destructor {
     /// The def-id of the destructor method
     pub did: DefId,
```

Comment:
I want to be more clear that this is the method def-id, rather than the impl-defid.